src/members/grid-list-item.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { memberShape } from './model';
+
+export default function MembersGridListItem(props) {
+  const { memberInfo } = props;
```

Comment:
We moved all props destructuring info the function in other files.

So it would be `function MembersGridListItem({ memberInfo })`.

Reply:
this actually break the code i don't  remember moving destructuring info to other files ?
this is how we did it in cases and posts components please can you check once again if you have time thanks :)

Reply:
sorry fixed thanks for the help :)